src/chocolatey.console/Program.cs

```diff
@@ -44,6 +44,8 @@ namespace chocolatey.console
     using Console = System.Console;
     using Environment = System.Environment;
     using IFileSystem = infrastructure.filesystem.IFileSystem;
+    using Microsoft.Win32;
+    using chocolatey.infrastructure.platforms;

```

Comment:
Can we order these so that there are in alphabetical order?

Reply:
Reordered